tests/ltp/buildenv.sh

```diff
@@ -45,6 +46,10 @@ if [[ "$mode" == "build" ]]; then
         git apply $FORK_DISABLE_PATCH
     fi
 
+    if [ -f $UMOUNT02_PATCH ];then
```

Comment:
Why would the patch file be missing?

Reply:
Chance is less but if the patch file is not copied to the proper path we can find it here during the build.

Reply:
Sorry I don't understand, could you explain it a bit more?

Reply:
I meant copying patch to mount point is done in Makefile, So thought its better to check here that is there. 
vamsi@vm-gshanker:~/ashok_latest/fork3/sgx-lkl/tests/ltp$ grep patch Makefile
        $(ESCALATE_CMD) cp ltp_fork_disable.patch $(MOUNTPOINT)/
        $(ESCALATE_CMD) cp patches/umount03.patch $(MOUNTPOINT)/

Reply:
So if the patch is not there then you don't apply it and continue happily as if everything is ok? Doesn't make much sense to me. Or am I missing something?

Reply:
Agree that it is not really required. Will remove the check.